src/lib/runQuickJs.ts

```diff
@@ -1,9 +1,123 @@
-import { getQuickJS } from 'quickjs-emscripten'
+import { getQuickJS, QuickJSContext } from 'quickjs-emscripten'
+
+import { HttpVerb} from 'then-request'
+import request from 'sync-request'
+
+function isHexString(str: string): boolean {
+  const regex = /^0x[0-9a-f]+$/
+  return regex.test(str.toLowerCase())
+}
+
+const hexToString = (hex: string): string => {
+  return Buffer.from(hex.substring(2), 'hex').toString()
+}
+
+function syncRequest(options: {
+  url: string,
+  method?: HttpVerb,
+  headers?: Record<string, string>,
+  body?: string,
+  returnTextBody?: boolean,
+}) {
+  if (options.body && isHexString(options.body)) {
+    options.body = hexToString(options.body)
+  }
+  const res = request(options.method || 'GET', options.url, {
+    headers: options.headers,
+    body: options.body,
+  })
+  return {
+    statusCode: res.statusCode,
+    body: res.getBody(options.returnTextBody ? 'utf8' : '')
+  }
```

Comment:
We need test for the fail cases and check is it any potential exceptions not yet handle.

Reply:
It will throw error messages to the console by now

Reply:
We should ensure it's behavior same as real environment.